packages/core/src/types/provider.ts

```diff
@@ -267,6 +294,10 @@ export type CreateAssetArgs<TSource extends CreateAssetSourceType> = {
    * environments.
    */
   chunkSize?: number;
+  /**
+   * Sets the playback policy for all of the assets created.
+   */
+  playbackPolicy?: PlaybackPolicy;
```

Comment:
Hmm, I feel like the playback policy is more of an asset configuration than of a "asset creation process" configuration (like `chunkSize`). For example, each created asset here could have different gating configurations. Maybe they'd use the same webhook, but most likely the webhook context would be different for each of them (e.g. each one is an NFT with a different tokenId)

WDYT of moving this field to the `CreateAssetSourceBase` type instead?

Reply:
Yeah that makes sense to me!